src/pages/other-services/resources-and-toolkits/child-safe-resources.tsx

```diff
@@ -0,0 +1,237 @@
+          them to grow and develop in safe, caring environments free from abuse
+          and harm.
+        </p>
+        <p className="mb-4">
+          Starick respects the rights of the child and recognises the
+          significant role that mother’s/carer’s play and seeks their
+          involvement to support the safety and wellbeing of their child in the
+          context of family and domestic violence.
+        </p>
+        <p className="mb-4">
+          Starick’s developing suite of child safe policies is aligned to the{" "}
+          <a
+            href="https://www.unicef.org.au/united-nations-convention-on-the-rights-of-the-child"
+            className="underline"
+          >
+            United Nations Convention on the Rights of the Child
+          </a>
+          , the{" "}
+          <a
+            href="https://www.ccyp.wa.gov.au/media/3946/child-safe-organisations-wa-guidelines-updated-november-2019.pdf"
+            className="underline"
+          >
+            National Principles and Standards for Child Safe Organisations
+          </a>{" "}
+          and
+          <a
+            href="https://www.aracy.org.au/documents/item/700#:~:text=What%20is%20the%20Nest%3F,young%20person%20needs%20to%20thrive."
+            className="underline"
+          >
+            Australia’s Wellbeing Framework for Children and Young People
+            What&apos;s in the Nest?
+          </a>{" "}
+          - Australian Research Alliance for Children and Youth (ARACY).
+        </p>
+        <p className="mb-4">
+          The National Child Safe Principles and Standards provide a framework
+          for creating child safe organisations. They are designed to create,
+          maintain, and improve child safe practices. The Standards are based on
+          extensive research and consultation by the Royal Commission into
+          Institutional Responses to Child Sexual Abuse. They provide clear
+          guidance for organisations to create cultures, adopt strategies and
+          act to put the interests of children first to keep them safe from
+          harm.
+        </p>
+        <p>
+          Below are some child safe policy and procedure templates that can be
+          downloaded and customised for organisations’ specific needs. If you
+          have any child safe policies or procedures that your organisation
+          would like to share, please email{" "}
+          <a href={`mailto:${emailAddress}`} className="underline">
+            info@starick.org.au
+          </a>
+          .
+        </p>
+      </div>
+      <div className="px-10 md:px-20 lg:px-32">
+        <SectionTitle
+          text="Child Safe Template"
+          backgroundColour="[--starick-green]"
+          textColour="[--starick-white]"
+          iconPath="/images/ResourcesToolkits/child-safe-templates-icon.png"
+        />
+      </div>
+
+      <div className="px-20 py-10 md:px-40 md:py-20 lg:px-60">
+        <ChildSafeItem
```

Comment:
I think this code could be refactored to better adhere to the DRY (don't repeat yourself) principle. This could be done by having a json object which stores the specific data for each row and then you use the js map function to create a loop over all the rows. See [here](https://react.dev/reference/react/Children#children-map) for how to do this.